browser/react/index.js

```diff
@@ -19,8 +23,18 @@ const onHomeEnter = () => {
 ReactDOM.render(
   <Provider store={store}>
     <Router history={browserHistory}>
-      <Route path='/' component={App} onEnter={onHomeEnter} />
-      <Route path='/login' component={Login} />
+      <Route path='/' onEnter={onHomeEnter} >
+        <IndexRoute component={Login} />
+        <Route path='/vr' component={App} onEnter={onHomeEnter} >
```

Comment:
I don't think we do. Probably just overlooked deleting it when I created the root element.